src/Admin/AbstractAdminExtension.php

```diff
@@ -96,6 +96,15 @@ public function configureBatchActions(AdminInterface $admin, array $actions): ar
         return $actions;
     }
 
+    // NEXT_MAJOR: Remove the PHPDoc block as the interface will then specify the types
```

Comment:
I don't understand this comment, to remove the PHPDoc here , PHP will need to support generics, and that it will not happen anytime soon, can you remove it?

Reply:
The phpdoc will be in the interface, so we won't be able to duplicate it in the class